snekbox/nsjail.py

```diff
@@ -24,16 +24,17 @@
 CGROUP_MEMORY_PARENT = Path("/sys/fs/cgroup/memory/NSJAIL")
 
 NSJAIL_PATH = os.getenv("NSJAIL_PATH", "/usr/sbin/nsjail")
+NSJAIL_CFG = os.getenv("NSJAIL_CFG", "./snekbox.cfg")
 MEM_MAX = 52428800
 
 
 class NsJail:
     """
     Core Snekbox functionality, providing safe execution of Python code.
 
-    NsJail configuration:
+    Default NsJail configuration (snekbox.cfg):
 
-    - Root directory is mounted as read-only
+    - All mounts are read-only
     - Time limit of 2 seconds
```

Comment:
If I'm not mistaken, @jos-b changed the time_limit to 5 seconds in his commit.
```suggestion
    - Time limit of 5 seconds
```